src/api_call.js

```diff
@@ -1,25 +1,23 @@
 //@format
 
+import GLOBALS from './globals.js';
+
 export async function generateSearchResults(pokeType) {
-  const typeSet = await getTypeSet(pokeType);
-  const filteredTypeSet = filterPokeIds(typeSet);
-  const pokeStatsPromises = filteredTypeSet.map(getPokemonStats);
-  const resolvedPokeStats = await Promise.all(pokeStatsPromises);
+  const filteredTypeSet = filterPokeIds(await getTypeSet(pokeType));
+  const resolvedPokeStats = await Promise.all(
+    filteredTypeSet.map(getPokemonStats),
+  );
   return resolvedPokeStats.map(filterCharacteristics);
 }
 
 async function getTypeSet(pokeType) {
-  const response = await fetch(
-    'https://pokeapi.co/api/v2/type/' + pokeType + '/',
-  );
-  return response.json();
+  return await (await fetch(GLOBALS.APIHOST + 'type/' + pokeType + '/')).json();
 }
 
 async function getPokemonStats(pokeId) {
-  const response = await fetch(
-    'https://pokeapi.co/api/v2/pokemon/' + pokeId + '/',
-  );
-  return response.json();
+  return await (await fetch(
```

Comment:
Hi, this is actually the way I had it originally - I changed it to (await (await...)) because I wanted to see if it would work and it does! Maybe I will change it back so it is clearer, though.
and thanks I will make sure to include the error handling!